debian/changelog

```diff
@@ -1,3 +1,9 @@
+containerd.io (1.6.31-2) release; urgency=medium
+
+  * update containerd binary to v1.6.31.
+
+ -- Paweł Gronowski <pawel.gronowski@docker.com>  Fri, 08 Apr 2024 13:31:20 +0000
```

Comment:
Should I adjust the date? 🤔

Reply:
I guess? The date matches the commit though 👀